document-portal/document-portal.c

```diff
@@ -775,7 +780,7 @@ document_add_full (int                      *fd,
                    int                      *parent_dev,
                    int                      *parent_ino,
                    int                       n_args,
-                   DocumentAddFullFlags      flags,
+                   DocumentAddFullFlags     *the_flags,
```

Comment:
I think I'd prefer if this variable was called `documents_flags`, or `add_full_flags`. Thoughts?

Reply:
`documents_flags` is a good idea.